### DISCUSSION TITLE
QTH / Location question

### DISCUSSION THREAD

**Discussion by timgrossner** [magicbug/Cloudlog#1413] [Q&A]
For QSOs that I am working, when it looks them up, its defaulting everyone in the USA to being in Milwaukee, WI. UK callsigns are defaulting to London, etc. Is there something I am missing that would cause this? I have the usernames and passwords for both HamQTH and QRZ.com in place.

Thanks, and 73.

Tim

**Comment by magicbug** [comment]
If you don't provide a gridsquare then the contact will default to the DXCC Lat/Long which is stored in the database which we get from Clublog.

If you have configured the config.php to use either QRZ or HamQTH then it should auto populate the fields in the QSO entry window when it manages to find a match.

Peter.

**Reply by timgrossner** [reply]
Sorry, just to be clear - if "you" don't provide a gridsquare - as in me putting their gridsquare in - just making sure I understand. Wouldn't the gridsquare come from the QRZ or HamQTH lookups of their callsign? Or am I misunderstanding?

**Reply by magicbug** [reply]
It should auto populate from one of the callbook services "if" the callsign is in their system.

Good example is me "2M0SQL" if that doesn't populate from QRZ.com you have issues with your configuration.

Peter.

**Comment by timgrossner** [comment]
yeah, something is wrong. Not sure what. What's the best way to debug/log it? I don't see a log file offhand for Cloudlog, other than the normal access.log and error.log for Apache.
